lib/rest2/methods.js

```diff
@@ -87,7 +88,29 @@ const RESTV2_MOCK_METHODS = {
   '/v2/auth/r/ext/pay/invoices': 'invoice_list',
   '/v2/auth/w/ext/pay/invoice/complete': 'invoice_complete',
   '/v2/auth/r/ext/pay/deposits/unlinked': 'deposits_unlinked',
-  '/v2/auth/r/movements/info': 'movement_info.{id}'
+  '/v2/auth/r/ext/pay/settings/convert/list':'pay_settings_convert_list',
+  '/v2/auth/w/ext/pay/settings/convert/create':'pay_settings_convert_create',
+  '/v2/auth/w/ext/pay/settings/convert/remove':'pay_settings_convert_remove',
+  '/v2/auth/r/ext/pay/settings/daily/limit':'pay_settings_daily_limit',
+  '/v2/auth/w/ext/pay/settings/set':'pay_settings_set',
+  '/v2/auth/w/ext/pay/settings/set/batch':'pay_settings_set_batch',
+  '/v2/auth/r/ext/pay/settings/get':'pay_settings_get',
+  '/v2/auth/r/ext/pay/settings/list':'pay_settings_list',
+  '/v2/auth/w/ext/pay/invoice/refund':'pay_invoice_refund',
+  '/v2/auth/w/ext/pay/invoice/mark/refunded':'pay_invoice_mark_refunded',
+  '/v2/auth/r/ext/pay/invoice/events':'pay_invoice_events',
+  '/v2/auth/w/ext/pay/instant':'pay_instant',
+  '/v2/auth/w/ext/pay/upload/image':'pay_upload_image',
+  '/v2/auth/r/ext/pay/credit-card/rates':'pay_credit_card_rates',
+  '/v2/ext/pay/invoice/currency/detailed':'pay_public_invoice_currency_detailed',
+  '/v2/auth/r/ext/pay/currency/detailed':'pay_currency_detailed',
+  '/v2/auth/r/ext/pay/currency/list':'pay_currency_list',
```

Comment:
there are linting issues, see missing spaces in `:`

Reply:
fixed!